index.html

```diff
@@ -548,7 +548,9 @@ <h2>
       </pre>
       <p>
         The <dfn>OrientationLockType</dfn> enum represents the screen
-        orientations to which a screen can be rotated.
+        orientations to which a screen can be potentially locked to. [=User
+        agents=] are NOT REQUIRED to support locking the screen orientation to
+        any of the values in the {{OrientationLockType}} enum.
```

Comment:
This should probably be stated in the algorithm belonging to `lock()`? At least this seems like a non-normative paragraph explaining the enum without relevance to the processing model.